README.md

```diff
@@ -17,7 +17,35 @@ Use this orb to load secrets from SecretHub into your CircleCI jobs.
 
 ## Usage
 
-To execute a command with secrets loaded from SecretHub, install the CLI and set it as the `shell` of the `run` command:
+To execute a command that needs secrets, replace your CircleCI `run` command with `secrethub/exec`.
+You can make secrets available to your command as environment variables by referencing their SecretHub path, prefixed by `secrethub://`:
+
+```yml
+version: 2.1
+orbs:
+  secrethub: secrethub/cli@v0.1.0
+
+jobs:
+  deploy:
+    docker:
+      - image: cimg/base:stable
+    environment:
+      AWS_ACCESS_KEY_ID: secrethub://company/app/aws/access_key_id
+      AWS_SECRET_ACCESS_KEY: secrethub://company/app/aws/secret_access_key
+    steps:
+      - checkout
+      - secrethub/exec:
+          command: |
+            echo "This value will be masked: $AWS_ACCESS_KEY_ID"
+            echo "This value will be masked: $AWS_SECRET_ACCESS_KEY"
```

Comment:
Do we want to explain this in the example code or explain it in text somewhere?

It now makes the example a bit more complex.

Reply:
Yes, we have the `Masking` section for that ⬇️

Reply:
Oke, so shall we remove it from the examples to make them clearer?

Reply:
No, it also fulfills the function of possibly preventing a major wtf-moment when seeing an `echo` of an `AWS_SECRET_ACCESS_KEY`.